src/simd_256.h

```diff
@@ -57,28 +57,26 @@ typedef __m128i HalfVecType;
 // with static storage duration may throw an exception that cannot be caught
 
 // NOLINTNEXTLINE(cert-err58-cpp)
-const VecType F4_U32 = _mm256_set1_epi32(65537);
+const VecType vec32_f4 = _mm256_set1_epi32(65537);
```

Comment:
I think you should keep the UPPERCASE naming.

They're not #define anymore, but they are still global constants and as entities with a global scope they should be in UPPERCASE.